tests/PHPUnit/Controller/ExportDataControllerTest.php

```diff
@@ -22,12 +22,14 @@ public function testExportDataWithCart(): void
         $client->loginUser($shopUser);
 
         // go to product page
-        $client->request('GET', '/en_US/products/sport-basic-white-t-shirt');
-        $this->assertSelectorTextContains('h1', 'Sport basic white T-Shirt');
+        $client->request('GET', '/en_US/products/solar-echo-t-shirt');
+        $this->assertSelectorTextContains('h1', 'Solar Echo T-Shirt');
 
         // add product to cart
         $client->submitForm('Add to cart');
+
         $client->request('GET', '/en_US/cart/');
+
         $this->assertSelectorTextContains('h1', 'Your shopping cart');
```

Comment:
it seems h1 no longer exist in v2
```suggestion
        $this->assertPageTitleContains('Your shopping cart');
```